README.md

```diff
@@ -1,5 +1,6 @@
 [![CircleCI](https://circleci.com/gh/sul-dlss/dlme.svg?style=svg)](https://circleci.com/gh/sul-dlss/dlme)
 [![Coverage Status](https://coveralls.io/repos/github/sul-dlss/dlme/badge.svg)](https://coveralls.io/github/sul-dlss/dlme)
+![Supported Ruby Version](https://img.shields.io/badge/dynamic/yaml?color=red&label=ruby&query=%24.parameters.ruby_version.default&url=https%3A%2F%2Fraw.githubusercontent.com%2Fsul-dlss%2Fdlme%2Fdocument-ruby-version%2F.circleci%2Fconfig.yml)
```

Comment:
```suggestion
![Supported Ruby Version](https://img.shields.io/badge/dynamic/yaml?color=red&label=ruby&query=%24.parameters.ruby_version.default&url=https%3A%2F%2Fraw.githubusercontent.com%2Fsul-dlss%2Fdlme%2Fmain%2F.circleci%2Fconfig.yml)
```

Presumably we'd want to point this at `main`, not this branch?

Reply:
oooh good catch, thx